src/components/ComputeConnectDialog.vue

```diff
@@ -259,12 +266,18 @@ config = {
     "spark.sql.catalog.${props.warehouse.name}.type": "rest",
     "spark.sql.catalog.${props.warehouse.name}.uri": "${icebergCatalogUrlSuffixed}",
     "spark.sql.catalog.${props.warehouse.name}.warehouse": "${props.warehouse.name}",
-    "spark.sql.catalog.${props.warehouse.name}.token": "${user.access_token}",
+    ${enabledAuthorization ? `"spark.sql.catalog.${props.warehouse.name}.token": "${user.access_token}",` : '##'}
```

Comment:
I think we should rename this in a next step. If it's used here, it should be called `enabledAuthentication`. If Authorization is disabled and Athentication enabled we still need the token. Currently the
* `enabledAuthorization` holds information about Authentication and
* `enablePermissions` about Authorization

Reply:
agree , same to .env VITE_ENABLE_AUTHORIZATION=true. Is more about Authentication. Authorization is then provided by serverInfo API